currencyii/src/main/java/nl/tudelft/trustchain/currencyii/ui/bitcoin/DAOLoginChoiceFragment.kt

```diff
@@ -39,11 +39,13 @@ class DAOLoginChoiceFragment : BaseFragment(R.layout.fragment_dao_login_choice)
         }
 
         // Make sure to hide any other wallets that exists, when creating a new wallet
+        // TODO: Is this even still in use?
```

Comment:
I think this should be checked before merging and removed

Reply:
I agree, but I can't find a page where it is still used, so I think it is not, maybe we can ask it to Johan